providers/jackson2/src/main/java/org/jboss/resteasy/plugins/providers/jackson/PatchMethodFilter.java

```diff
@@ -80,23 +94,43 @@ public void filter(ContainerRequestContext requestContext) throws IOException
          catch (ProcessingException pe)
          {
             Throwable c = pe.getCause();
-            if (c != null && c instanceof ApplicationException) {
+            if (c != null && c instanceof ApplicationException)
+            {
                c = c.getCause();
-               if (c != null && c instanceof NotFoundException) {
-                  throw (NotFoundException)c;
+               if (c != null && c instanceof NotFoundException)
+               {
+                  throw (NotFoundException) c;
                }
             }
             throw pe;
          }
-         catch (JsonMappingException | JsonParseException e) {
+         catch (JsonMappingException | JsonParseException e)
+         {
             throw new BadRequestException(e);
          }
          catch (JsonPatchException e)
          {
             throw new Failure(e, HttpResponseCodes.SC_CONFLICT);
          }
+
       }
 
    }
 
+   private Object getTarget() throws ProcessingException
+   {
+      HttpRequest request = ResteasyContext.getContextData(HttpRequest.class);
+      HttpResponse response = ResteasyContext.getContextData(HttpResponse.class);
```

Comment:
You should probably pass 'request' and 'response' objects here from the caller as parameters instead of getting them again from the context here